MANIFEST.in

```diff
@@ -3,3 +3,6 @@ include *.rst
 include *.toml
 include LICENSE
 include tox.ini
+
+prune .github
+prune tests
```

Comment:
The tests were included before, let's keep them. Often downstream distro packagers want them included so they can also test, and they're very small so not a problem to keep.

```suggestion
```